seeds/database_setup.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TABLE IF NOT EXISTS users (id SERIAL PRIMARY KEY, email TEXT, username TEXT, password TEXT);
```

Comment:
i think it would be nice if you added some keywords to. your column definitions such as NOT NULL or use a bigger range of data types to reflect the use/necessity of the data better. Maybe make use of some default values too to make it easier on yourself in the long run such as for post_date.